### ISSUE TITLE
dynamic backlight does not work (AN515-46)

### ISSUE THREAD

**Issue by FauxJD** [JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module#118]
# Model: NITRO AN515-46

### RGB keyboard works?  Yes

**Comment by FauxJD** [comment]
only this command "./facer_rgb.py -m 0 -z 4 -cR 255 -cB 255 -cG 0" works fine

**Comment by JafarAkhondali** [comment]
Can you uninstall & install again the project?

**Comment by FauxJD** [comment]
I'm trying this now. But the dynamic mode did not work (only changing the static mode works). And I didn't tell you that I have an LED indicator on the F8 button. Could we be talking about another revision of the keyboard?

**Comment by JafarAkhondali** [comment]
Yes maybe your keyboard is completely different, i thought the recent update might fix your issue

**Comment by JafarAkhondali** [comment]
Thanks for the update. Did turbo mode work for you?

**Comment by FauxJD** [comment]
My keyboard doesnt have a turbo-boost button (if we are speak about fans), only have "nitro sence" button, indicators of microphone (It can normal works only in win11, in linux it all time glowing, but microphone is works fine ) and caps-lock.
